var/spack/repos/builtin/packages/dsrc/package.py

```diff
@@ -0,0 +1,46 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Dsrc(MakefilePackage):
+    """DNA Sequence Reads Compression is an application designed for
+    compression of data files containing reads from DNA sequencing in
+    FASTQ format."""
+
+    homepage = "http://sun.aei.polsl.pl/dsrc"
+    url      = "https://github.com/refresh-bio/DSRC/archive/v2.0.2.tar.gz"
+
+    version('2.0.2',     '0a75deb6db948f9179df0756c259b870')
+
+    parallel = False
+
+    def install(self, spec, prefix):
+        make()
```

Comment:
You can get rid of this line, it is run for you during the `build` phase already.

Reply:
makes sense!